src/components/ContentWrapper.tsx

```diff
@@ -44,6 +44,7 @@ const ContentWrapper = ({ children }) => {
         "profile",
         "supportive-materials",
         "search-page",
+        "sources",
```

Comment:
question: Should we inver the logic now? Most of the pages are desktop ready and maybe it makes more sense to track the ones that aren't